### PR TITLE
Fix npm-publish action

### PR DESCRIPTION
**Pull request by afflerbach**
* add missing env
* Fix token issue
* drop inputs
* hardcode URL (issue with older node versions)
* publish both packages

Describe your changes!

---
<details>
    <summary>Reviewer checklist</summary>
    <ul>
        <li>Read PR description: a summary about the changes is required</li>
        <li>Changelog updated</li>
        <li>Documentation: docs/{Reference, Cli, ...}, Docker and cli help/usage</li>
        <li>Pulled branch, manually tested</li>
        <li>Verified requirements are met</li>
        <li>Reviewed the code</li>
        <li>Reviewed the related tests</li>
    </ul>
</details>

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -3,49 +3,50 @@ name: npm-publish
...
-        description: Package name
-        type: choice
-        default: "couper"
-        required: true
-        options:
-        - "couper"
-        - "@avenga/couper"
       package-version:
-        description: Package version
-        required: true
```
Comment (malud) [inline .github/workflows/npm-publish.yml:16]: Still required?

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -3,49 +3,50 @@ name: npm-publish
 on:
   workflow_dispatch:
     inputs:
-      package-name:
-        description: Package name
-        type: choice
-        default: "couper"
-        required: true
-        options:
```
Comment (malud) [inline .github/workflows/npm-publish.yml:11]: Lets add the matrix probs here:

```yaml
strategy:
      matrix:
        pkg: ['couper', '@avenga/couper']
```

and read from `${{ matrix.pkg }}` later on

File: .github/workflows/npm-publish.yml [hunk 3]
```diff
@@ -3,49 +3,50 @@ name: npm-publish
...
+      PACKAGE_VERSION: ${{ github.event.inputs.package-version || github.ref_name }}
+      # https://github.com/JS-DevTools/npm-publish/issues/15
+      INPUT_TOKEN: ''
     steps:
       - name: 'checkout'
         uses: actions/checkout@v2
       - name: 'prepare package'
         run: |
           cp LICENSE .npm
+          cp -r .npm .npm2
```
Comment (malud) [inline .github/workflows/npm-publish.yml:25]: As said above, matrix would be more readable

File: .npm/couper-binary.js [hunk 4]
```diff
@@ -34,15 +34,16 @@ function getPlatform() {
 class CouperBinary {
 
 	constructor() {
-		// FIXME
-		let { version, name, repository } = require("./package.json")
+		let { version } = require("./package.json")
 		version = "1.7.1"
```
Comment (malud) [inline .npm/couper-binary.js:38]: seems to be hardcoded ?

**Review by malud — CHANGES_REQUESTED** [review]

File: .github/workflows/npm-publish.yml (hunk 2 above)
Comment (afflerbach) [reply .github/workflows/npm-publish.yml:11]: :star_struck:

File: .npm/couper-binary.js (hunk 4 above)
Comment (afflerbach) [reply .npm/couper-binary.js:37]: Yeah, until we get in sync with the package version. :unamused:

File: .github/workflows/npm-publish.yml (hunk 3 above)
Comment (afflerbach) [reply .github/workflows/npm-publish.yml]: Done!

**Review by malud — APPROVED** [review]

**Pull request merged**